snapcraft/projects.py

```diff
@@ -180,17 +180,38 @@ def _validate_list_stream(cls, listen_stream):
         return listen_stream
 
 
-class LintIgnore(ProjectModel):
-    """Lists of specific linters or globbed filenames to ignore."""
+class Lint(ProjectModel):
+    """Linter configuration.
+
+    :ivar ignore: A dict describing which files should have issues ignored for given linters.
+        The keys in the dict are the names of known linters (currently "library" and "classic"):
```

Comment:
for maintenance, it might be better to make https://github.com/tigarmo/snapcraft/blob/38d3b738b3d946db4799092b71ffb71256cb6d0f/snapcraft/linters/linters.py#L43 public and refer to that here (instead of the "currently" list)